openshift-ci/ztp-left-shifting/sno-hub/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM registry.ci.openshift.org/ci/telco-runner
+
+RUN pip3 install --no-cache-dir "pytest==8.2.2" "pytest-shell==0.3.2" "requests==2.32.3"
```

Comment:
If that's the only change, maybe we can add it to the `telco-runner` image?

Reply:
I would prefer not to, as I believe this will only be a temporary thing. The most correct way to run the checks here will be using Go-test, but for now I was just focusing on placing the various pieces in their then. This Dockerfile will probably need to be updated later so that the Go libraries to be used are installed. As I say, this PR is more to analyze and understand how to call this from the Openshift-CI job.